deploy/helm-chart/README.md

```diff
@@ -105,6 +124,53 @@ the release `my-release`, run:
 helm upgrade my-release . -n nginx-gateway
 ```
 
+### Configure Delayed Termination for Zero Downtime Upgrades
```

Comment:
Should this section precede the upgrade instructions so users know they must have this configured for zero downtime upgrades?

Reply:
Not every user will want to configure this so that's why I added it after, but I'm happy to move it if we feel that makes more sense!

Reply:
I think it's probably worth having before, though I wonder how we handle config sections without causing too much clutter to the installation guides...

Reply:
I moved it up and then I thought it made the upgrade docs a bit hard to follow. What about adding a note and a link to the start of the upgrade section e.g.

```
> **Note**
> See [below](#configure-delayed-termination-for-zero-downtime-upgrades) for instructions on how to configure
> delayed termination if required for zero downtime upgrades in your environment.
```

Reply:
That sounds good to me.

Reply:
Be sure to add to both docs

Reply:
works for me!